Add wrapup blog post for PDX conference and update website (#1322)
writethedocs/www: docs/conf/portland/2020/news/thanks-recap.rst

:template: {{year}}/generic.html
:banner: _static/conf/images/headers/venue.png

.. post:: Aug 20, 2020
   :tags: recap, thanks, portland-2020

Conference highlights and thanks to the community!
--------------------------------------------------

Hey everyone,

The Write the Docs team is now emerging from the post-conference
recovery period, and we wanted to send out a quick message to thank
everyone for helping to make our 1st online conference such a
success. We're happy to share some of the highlights with you.

Conference recap
================

At over 600 people this year's conference was our biggest yet! We had
the usual mix of main stage and lightning talks, unconference sessions,

> writing sprints, and social events. We were very happy how well all our

Reviewer: ```suggestion
writing sprints, and social events. We were very happy with how well all of our
```

> events transitioned to our new online format. We also felt the spirit of the community 

Reviewer: ```suggestion
events transitioned to our new online format. We loved how the spirit of the community 
```

> come through the screen, which was even more special given the year we've all had.

Reviewer: ```suggestion
was so palpable even in an online event, which was even more special given the year we've all had.
```

Overall, we were excited with how our awesome community come together in our new venue.
Lots of knowledge was shared, and we had a great time.

Videos
======

Videos of the talks are now available on `our YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ>`__. Sadly, the Monday afternoon talks had an issue with the Q&A not being recorded, so we weren't able to add those to the videos. We think we know what happened with it, and hopefully we've addressed this issue going forward.

Thanks to `Bart at Work <https://www.bartatwork.com/atwork/>`__ for doing such a great job with these. You can also subscribe to our `YouTube channel <https://www.youtube.com/writethedocs>`__ so you don't miss any future content!

> 

Reviewer 2: ```suggestion
Shirts still on sale
====================

If you want a souvenir from this year's conference, our `Write the Docs Portland 2020 Pop-Up Shop <https://teespring.com/wtd-portland-2020>`_ is still open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches until August 30th. Shipping is available from the USA or Europe.

```

- The CoC, or a summary, was included on the conference website and Hopin Reception
- The CoC was repeatedly mentioned in our conference introduction and announcements.
- We specifically stressed that the CoC applies to all conference spaces including the social event.
- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.

None of our organizers were approached about any CoC concerns at this event.
We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.

That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened. If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behavior set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.

Ready for more?
===============

Don’t forget our `Prague conference <https://www.writethedocs.org/conf/prague/2020/>`__ in
October, and our `Australia+India conference <https://www.writethedocs.org/conf/australia/2020/>`__ in

> December. Tickets are on sale for Prague. The CFP and tickets for the Australia conference are open now.

Reviewer: ```suggestion
December. Tickets are on sale for Prague. The call for talk proposals for the Australia conference is still open, and ticket sales are open.
```

Reviewer 3: Let's  add something like: 
> Our Prague and Australia+India conferences are also virtual this year, so you can attend or present from anywhere. They take place during the day in the CEST and AEDT time zones, respectively.

> Our Slack network has really grown more this last year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, now's the time! Check out the `Write the Docs Slack <http://www.writethedocs.org/slack/>`__.

Reviewer: ```suggestion
Our Slack network has expanded yet again this year. Our #general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, now's the time! Check out the `Write the Docs Slack <http://www.writethedocs.org/slack/>`__.
```
